content/en-us/reference/engine/classes/RunService.yaml

```diff
@@ -110,7 +110,7 @@ methods:
         make a binding right before the default camera update, simply subtract 1
         from the camera priority level.
 
-      When using Enum.RenderPriority, remember to use **_InlineCode.Value_** at
+      When using Enum.RenderPriority, remember to use <InlineCode>.Value</InlineCode> at
```

Comment:
```suggestion
      When using Enum.RenderPriority, remember to use `.Value` at
```

Reply:
Not sure how that formatting snuck in there. Backticks will do the same thing here.